main.go

```diff
@@ -179,6 +185,9 @@ func Execute(cmd *cobra.Command, args []string) {
 	http.HandleFunc("/metrics/general", func(w http.ResponseWriter, r *http.Request) {
 		GeneralHandler(w, r, grpcConn)
 	})
+	http.HandleFunc("/metrics/stream", func(w http.ResponseWriter, r *http.Request) {
+		streamCollector.StreamHandler(w, r)
+	})
```

Comment:
nit, thoughts on `Events` instead? We're subscribing to events, it's not exactly a stream a data

Reply:
yeah sg, will rename